frontend/javascripts/test/model/binary/cube.spec.js

```diff
@@ -98,7 +100,7 @@ test("GetBucket should only create one bucket on getOrCreateBucket()", t => {
 
 test("Voxel Labeling should request buckets when temporal buckets are created", t => {
   const { cube, pullQueue } = t.context;
-  cube.labelVoxel([1, 1, 1], 42);
+  cube.labelVoxelInResolution([1, 1, 1], 42, 0);
```

Comment:
The zoomStep is the 3rd parameter and the cellId is the 4th. Same for the following calls.